src/components/input/select/style.css

```diff
@@ -33,7 +34,7 @@
 }
 :host().missing > aside {
 	background-color: rgb(var(--smoothly-warning-color));
-	color: rgb(var(--smoothly-warning-contast));
+	color: rgb(var(--smoothly-warning-contrast));
```

Comment:
should prefer text-color?

Reply:
I dont think so, This styling is for a warning and therefore it makes sense that it uses the smoothly-warning colors.

I think this is one of the cases where it makes sense to not use --smoothly-color and/or the color specified by the form.